src/steps/generic.rs

```diff
@@ -90,7 +90,7 @@ pub fn run_rubygems(base_dirs: &BaseDirs, run_type: RunType) -> Result<()> {
     print_separator("RubyGems");
 
     let mut command = run_type.execute(gem);
-    command.arg("update --system");
+    command.args(["update", "--system"]);
 
     if env::var_os("RBENV_SHELL").is_none() {
         debug!("Detected rbenv. Avoiding --user-install");
```

Comment:
Either I'm still cranky form my cold, or this line does not make sense. Or it's totally in a wrong place.

Reply:
This does make sense because this informant doesn't have to be displayed when run in default mode.

Reply:
@DottoDev I said the statement does not make sense because it's within the `if` block that's triggered when there is **no RBENV**, aka - no `RBENV_SHELL` env var set. More so, the statement says `**avoiding** --user-install`, while in fact `--user-install` **is** added to the command invocation.

Unless I'm misreading this code,  this block should look like
```rs
if env::var_os("RBENV_SHELL").is_none() {
        command.arg("--user-install");
        debug!("Did not detect rbenv. Adding --user-install");
} else {
        debug!("Detected rbenv. Avoiding --user-install");
}
```

Reply:
Indeed, the message logged seems misleading..
If you prefer, I can also mark this conversation as resolved and immediately open up a separate PR with what you suggested, @mouse07410 ..

Reply:
@mouse07410 you are right, it does the opposite of what it's supposed to do.

Reply:
> Indeed, the message logged seems misleading..
> 
> If you prefer, I can also mark this conversation as resolved and immediately open up a separate PR with what you suggested, @mouse07410 ..

Yes please! This would be the best course. Thank you!

Reply:
Marked as resolved for merging

Reply:
> > Indeed, the message logged seems misleading..
> > If you prefer, I can also mark this conversation as resolved and immediately open up a separate PR with what you suggested, @mouse07410 ..
> 
> Yes please! This would be the best course. Thank you!

Just created https://github.com/topgrade-rs/topgrade/pull/249 based on your suggestions.